batavia/modules/time.js

```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
```

Comment:
At column 36: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)